static/src/javascripts/projects/commercial/modules/comscore.js

```diff
@@ -24,47 +25,51 @@ const getGlobals = (consentState, keywords) => {
 	return globals;
 };
 
-const initOnConsent = (state) => {
```

Comment:
I've refactored `initOnConsent` and `getGlobals` so that they don't take a state parameter because they're only ever called if the value of that argument is true.